.nvmrc

```diff
@@ -1,2 +1,2 @@
-17.9.0
+22.5.1
```

Comment:
The Jenkinsfile is using node:20 images. I think the nvmrc should as well instead of running a newer version.

Reply:
The 20 image is just for the release stage that requires git. The test matrix includes 22 for testing the code, so it should be ok to call 22 the latest.